blueprint/comparison.md

```diff
@@ -191,7 +191,7 @@ Screenshot | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gui/issues/95
 View angles | ✓ | ✓
 Apply force / torque | ✓ |
 Visualize as transparent | ✓ |
-Visualize as wireframe | ✓ |
+Visualize as wireframe | ✓ | ✕ (available from Fortress)
```

Comment:
Note that we don't need to update the Blueprint comparison anymore since that's EOL